hphp/runtime/ext/collections/ext_collections.h

```diff
@@ -1,9 +1,53 @@
-#ifndef incl_HPHP_EXT_COLLECTIONS_H
-#define incl_HPHP_EXT_COLLECTIONS_H
+/*
```

Comment:
hphp/runtime/ext/ext_collections.{h,cpp} and hphp/runtime/ext/collections/ext_collections.{h,cpp} are different files on purpose.  Please don't combine them as part of this diff.

If you want to move any file from runtime/ext/ to runtime/ext/FOO/ and there's already a version of that file there, then please rename it to ext_FOO-idl.{h,cpp} otherwise you're going to make IDL->HNI conversions really ugly.